packages/runner/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "browser": "src/index.js",
   "scripts": {
+    "prebuild": "rm -rf ./dist",
```

Comment:
We probably want `rimraf` here, like on line 10, for windows compat.

Reply:
It shouldn't have an impact on CI since the directory won't exist on a fresh machine, but could pose problems for any contributor on a windows machine. added in 0342424287

Reply:
I also installed `rimraf` in the directory since it does have an explicit dependency on it